includes/admin/payments/orders.php

```diff
@@ -684,6 +684,29 @@ function edd_order_details_overview( $order ) {
 		}
 	}
 
+	$has_tax  = 0;
+	$tax_rate = $order->get_tax_rate();
+	if ( edd_is_add_order_page() && edd_use_taxes() ) {
+		$has_tax = array(
+			'rate'    => 0,
+			'country' => '',
+			'region'  => '',
+		);
+	} elseif ( $tax_rate ) {
+		$has_tax = array(
+			'rate'    => $tax_rate,
+			'country' => '',
+			'region'  => '',
+		);
+		if ( $order->tax_rate_id ) {
+			$tax_rate_object = edd_get_adjustment( $order->tax_rate_id );
```

Comment:
https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8253/files adds a new `get_tax_rate_object()` method so let's wait until that's merged, then this PR can be updated to use that.

Reply:
Oh, I can go ahead and update this PR--it's based off of that one. I did use the new `get_tax_rate` method in here already.

Reply:
Sure! :+1: